docs/Data Lake.md

```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
+
+![QC Scatter]()
+
+*   **Cell Visualization**
+
+    *Cell visualization* provides exploration of cells using dimensionality reduction methods. The tab presents, dimensionality reduction methods such as tSNE, UMAP, PCA and others to visualize the distribution of cell. The *Visualize cells* panel on the right, shows the distribution of the cells in an interactive way. 
+
+![]()    
+
+    Using *Feature selection* panel, a metadata or a gene feature can be selected for plotting. 
+
+![]()
+    
+    The *Customize visualization* panel, offers the scope of customizing visualization features such as highlight non-zero cells, point size and method used based on personal preferences.
+
+![]()       
+    
+*   **Marker**
+
+    *Marker* tab provides exploration of distribution of markers. The tab presents, visualization methods like dot and violin plots to study the distribution of genes across different metadata. The Marker selection panel, provides the user with options to choose different genes and metadata for plotting marker distribution. 
+
+![]()
+
+    The dot plot panel, is the area for exploring the average expression and distribution of marker using dot plot in an interactive fashion.
+    
+![]()
+    
+    The Violin plot panel is the area for exploring the marker distribution using violin plot in an interactive fashion. Using Customize Violin slider, the range of values used for plotting violin can be adjusted, it is useful for observing a section of data such as the non-zero values of the expression. 
+
+![]()
+
+
+*   **Cell-Type Aggregation**
+
+    After having insights about different markers in a single cell study, it is imperative to have a look at the expression of the selected markers across different cell types in different studies. Single cell visualization provides pre-dataset exloratory analysis in the form of Cell Type Aggregation. Cell Type Aggregation provides the scope to query the respective repository for median expression of a selected gene across top 20 cell types based on expression. This provides the scope to study expression of a gene in a particular cell type in a given biological context. 
+
+![]()
+    
+    Potential use-case example: What is the expression of SARS-CoV2 virus entry specific host protein 'ACE2' across different cell types in different studies? 
+
+![]()
+    
+    The Cell Type Aggregation tab provides the input in the form of text field where query can be made for a gene, using our internal discover services, the input gene is queried across all the datasets in which it is expressed in the repository. To check for ACE2 expression, simply enter ACE2 in the search field. After entering the gene, clicking on 'Search gene' will make a bar plot showing the median expression and the distribution of gene expression across different studies.
```

Comment:
search gene in italics... will generate* a bar plot

Reply:
Done